biorxiv/pca_association_experiment/biorxiv_pca_category_bootstrap.py

```diff
@@ -290,12 +291,14 @@
     + p9.scale_x_discrete(
         limits=(
             projected_documents_df
-            .sort_values("category")
+            .groupby("category")
```

Comment:
Is this boxplot showing the distribution of category = "genomics" along PC1? I'm curious what this is telling you?

Reply:
Or are these values the similarity of documents in category= "genomics" with PC1?

Reply:
> Is this boxplot showing the distribution of category = "genomics" along PC1? I'm curious what this is telling you?
>Or are these values the similarity of documents in category= "genomics" with PC1?

This boxplot is showing the distribution of cosine similarity scores between documents within the genomics category and pc1. Its showing the spread of similarity scores across categories.